src/js/InAppList.js

```diff
@@ -10,11 +10,16 @@ export default class InAppMenu extends React.Component {
         const state = store.getState()
         const app = state.ui[state.activeApp]
         var layout = "LIST"
-        if (app && !app.isPerformingInteraction) {
-            if (app.activeSubMenu) {
-                layout = SubmenuDeepFind(app.menu, app.activeSubMenu, 0).subMenu.menuLayout;
+        if (app) {
+            if (app.isPerformingInteraction) {
+                layout = (app?.interactionLayout == "ICON_ONLY" || app?.interactionLayout == "ICON_WITH_SEARCH") 
```

Comment:
the layout looks good, but the `initialInteractionText` is nowhere to be found, what do you think about showing that initial message as the template title while in a perform interaction?

Reply:
I think thats a good idea. Implemented in https://github.com/smartdevicelink/generic_hmi/pull/389/commits/9783bfd50d78587e3309f3366b942aede6579ce2